config.dev.js

```diff
@@ -14,10 +14,13 @@ module.exports = {
     verified_role_name: 'Verified',
     mod_role_name: 'Moderator',
     alumni_role_name: 'Alumni',
+    officer_role_name: 'ACM Officer',
+    officer_alumni_role_name: 'ACM Officer Emeritus',
+    pvp_role_name: 'PVP',
   },
   ses: {
     region: 'us-west-1',
-    sender: 'smahadev369@g.ucla.edu',
+    sender: 'acm@ucla.edu',
```

Comment:
Are we changing it to this now? Just checking if you meant to do it or if it was just a temporary change for testing

Reply:
Yes, changing it to acm email